mtb-trails-db.sql

```diff
@@ -0,0 +1,23 @@
+-- See design notes in db-design.txt
+
+CREATE TABLE TRAIL_DIFFICULTIES (
+	DIFFICULTY_ID		SMALLINT NOT NULL,
+	TYPE			VARCHAR(255 BYTE),
+	PRIMARY KEY 		(DIFFICULTY_ID)
+);
+
+CREATE TABLE TRAILS (
+	ID				INTEGER NOT NULL,
+	NAME				VARCHAR(255 BYTE),
```

Comment:
Why VARCHAR and not VARCHAR2? I'm good with either but I'm just curious

Reply:
The only reason I used VARCHAR is because it was listed as the SQL data type for character strings in the reference I was using:
http://www-db.deis.unibo.it/courses/TW/DOCS/w3school/sql/sql_datatypes_general.asp.html

After researching more I'll change it to VARCHAR2, don't really need any distinction between "" and NULL.